src/gui/symbology-ng/qgsellipsesymbollayerv2widget.cpp

```diff
@@ -109,6 +109,45 @@ void QgsEllipseSymbolLayerV2Widget::setSymbolLayer( QgsSymbolLayerV2* layer )
   mHorizontalAnchorComboBox->setCurrentIndex( mLayer->horizontalAnchorPoint() );
   mVerticalAnchorComboBox->setCurrentIndex( mLayer->verticalAnchorPoint() );
   blockComboSignals( false );
+
+  mDataDefinedPropertyButtons.clear();
+
+#define QGS_REGISTER_DATA_DEFINED_BUTTON( button, name, type, description )\
```

Comment:
@vmora I don't think this should be a macro - I can't see any benefit in implementing it this way, and it breaks things like Qt Creator's find/follow/etc. I'd prefer that this was a function in QgsSymbolLayerV2Widget.